tests/test_common.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+from tern.analyze.common import add_package_files
+
+
+class TestCommon(unittest.TestCase):
+    def testAddPackageFiles(self):
+        a_pkg = {
+            'files': '77304005ceb5f0d03ad4c37eb8386a10866'
+                     'e4ceeb204f7c3b6599834c7319541 /usr/'
+                     'include/zconf.h|-rw-r--r-- 1 1000 '
+                     '1000 16262 Nov 13 17:57 /usr/include/zconf.h\n'
```

Comment:
It looks like the expectation is that the output of the script being invoked would be of the form `checksum file_path|permissions links UID GID size ISO date filepath\n`. It looks like the `add_package_files` assumes that the list is of this form. There are a couple of issues here:
* `extattrs` property includes the date and we have a `date` property in FileData that must be of the form `'%Y-%m-%d'`
* fs_hash extracts extended attributes like `permissions|uid|gid|size in bytes|number of hard links|` which is different from this form